server/src/main/java/org/elasticsearch/common/geo/GeoFormatterFactory.java

```diff
@@ -16,24 +16,24 @@
 import java.util.function.Function;
 
 /**
- * Output formats supported by GeoFields.
+ * Output formatters supported by geo fields.
  */
-public class GeoFormatsFactory {
+public class GeoFormatterFactory {
 
     public static final String GEOJSON = "geojson";
```

Comment:
Maybe we should move these constants into the same GeometryFormat file?

Reply:
I am going to leave it as it is for now. Those names refers to the name of the formatters which happens to be called the same between geo and cartesian shapes. Still the might be different and the current layout enforces that.